src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.props

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition="'$(EnableDotnetAnalyzers)' == 'true'">
+    <!-- Enable the Microsoft.Dotnet.CodeAnalysis.dll analyzer --> 
+    <Analyzer Include="$(BuildToolsTaskDir)Microsoft.Dotnet.CodeAnalysis.dll"/>
+
+    <AdditionalFiles Include="$(MSBuildProjectDirectory)\*.analyzerdata" />
```

Comment:
We have none yet. I included it as a quick way to pass data to the analyzer. I will refine this later on.